stan/math/prim/fun/add.hpp

```diff
@@ -8,6 +8,21 @@
 namespace stan {
 namespace math {
 
+/**
+ * Return the sum of the specified scalars.
+ *
+ * @tparam ScalarA Type of the first scalar.
```

Comment:
[optional]
The doc strings don't need caps or periods because they're not full sentences.